browser-test/browser-test-compose.dev.yml

```diff
@@ -13,7 +13,7 @@ services:
     tty: true # docker run -t
     ports:
       - 9457:9457
-    command: -jvm-debug "*:9457" ~run -Dconfig.file=conf/application.dev.conf
+    command: -jvm-debug "*:9457" ~run -Dconfig.file=conf/application.dev-browser-tests.conf
```

Comment:
Not blocking: What about `application.browser_tests.conf`?

Reply:
I don't see a clear pattern on the two and the directory uses -